projects/media-viewer/src/lib/toolbar/main-toolbar/main-toolbar.component.html

```diff
@@ -11,11 +11,13 @@
         <div id="mvToolbarMoreOptions" class="mv-toolbar-more-options">
           <button
             id="mvMoreOptionsBtn"
-            class="mv-button mv-toolbar__menu-button--more-options"
+            class="mv-button mv-toolbar__menu-button--more-options tooltip"
             [class.mv-toolbar__menu-button--more-options__hidden]="
               mvToolbar.offsetWidth >= allButtonsWidth
             "
             [attr.aria-expanded]="isDropdownMenuOpen"
+            data-tooltip="More options"
+            aria-label="More Options"
```

Comment:
Very small issue, but we always use Sentence case, not Title Case, i.e. should be "More options" not "More Options"